tests/formats/autodetect/bzng-gz.out.zng

```diff
@@ -1,2 +1,2 @@
-#0:record[_path:string,ts:time,uid:string,id:record[orig_h:addr,orig_p:port,resp_h:addr,resp_p:port],version:string,cipher:string,curve:string,server_name:string,resumed:bool,last_alert:string,next_protocol:string,established:bool,cert_chain_fuids:vector[string],client_cert_chain_fuids:vector[string],subject:string,issuer:string,client_subject:string,client_issuer:string,validation_status:string]
+#0:record[_path:bstring,ts:time,uid:bstring,id:record[orig_h:addr,orig_p:port,resp_h:addr,resp_p:port],version:bstring,cipher:bstring,curve:bstring,server_name:bstring,resumed:bool,last_alert:bstring,next_protocol:bstring,established:bool,cert_chain_fuids:vector[bstring],client_cert_chain_fuids:vector[bstring],subject:bstring,issuer:bstring,client_subject:bstring,client_issuer:bstring,validation_status:bstring]
```

Comment:
why is _path a bstring here but a string in some other `.zng`s?

Reply:
For cases other than this, I left the automatically added `_path` fields as type `string` (sans b).  But in this case, `_path` was explicitly declared in the bzng file as type bstring and I just updated the test expectations rather than changing the input file.
I don't really have any principled reasoning for all this.  In general, our test coverage is thin (i.e., we should have input files that exercise all the different types in bzng), I guess the real design question is what zng type an auto-added `_path` field should have (string vs bstring)